.github/workflows/e2ePerformanceTests.yml

```
@@ -4,117 +4,139 @@ on:
   workflow_call:
     inputs:
       PR_NUMBER:
-        description: A PR number to run performance tests against. If the PR is already merged, the merge commit will be used. If not, the test merge commit will be used.
+        description: A PR number to run performance tests against. If the PR is already merged, the merge commit will be used. If not, the PR will be merged locally before running the performance tests.
         type: string
         required: true
 
   workflow_dispatch:
     inputs:
       PR_NUMBER:
-        description: A PR number to run performance tests against. If the PR is already merged, the merge commit will be used. If not, the test merge commit will be used.
+        description: A PR number to run performance tests against. If the PR is already merged, the merge commit will be used. If not, the PR will be merged locally before running the performance tests.
         type: string
         required: true
 
 jobs:
-  prepare:
+  buildBaseline:
     runs-on: ubuntu-latest
-    name: Prepare to run builds
-    outputs:
+    name: Build apk from latest release as a baseline
```

```suggestion
  buildBaseline:
    runs-on: ubuntu-20.04-64core
    name: Build apk from latest release as a baseline
```
Modified for testing purposes, revert change before merging to `main`.

---

Bump, need to change back to `ubuntu-20.04-64core`